src/platform/amd/common/include/platform/ipc.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 AMD. All rights reserved.
+ *
+ *Author:       Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+ *		SaiSurya, Ch <saisurya.chakkaveeravenkatanaga@amd.com>
+ */
+#ifndef _COMMON_IPC_HEADER
+#define _COMMON_IPC_HEADER
+
+#include <stdint.h>
+#include <platform/platform.h>
+
+inline uint32_t sof_ipc_host_status(void);
+inline uint32_t sof_ipc_host_ack_flag(void);
+inline void sof_ipc_host_ack_clear(void);
+inline uint32_t sof_ipc_host_msg_flag(void);
+inline void sof_ipc_host_msg_clear(void);
+inline void sof_ipc_dsp_ack_set(void);
+inline uint32_t sof_ipc_dsp_status(void);
+inline void sof_ipc_dsp_msg_set(void);
+void irq_handler(void *arg);
```

Comment:
> 
Since this function is declared and used in IPC specific functions. we feel this name is ok as we can understand that this is used for IPC functionality.

Reply:
Updated the `irq_handler` to `amd_irq_handler`.